docs/devguide/quickstart/shapes/reliability.rst

```diff
@@ -0,0 +1,36 @@
+###########
+Reliability
```

Comment:
It might be good to link to the relevant sections in the devguide for further reading like in the introduction(s) or in the QoS sections. The [`seealso` directive](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-seealso) is good for this sort of thing.